### PR TITLE
Implement find-window, open and close on Linux (part of #12)

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin @georgiy-abbasov

### CODE REVIEW

File: Gulpfile.js [hunk 1]
```diff
@@ -88,6 +90,29 @@ gulp.task('copy-mac-scripts', ['clean-mac-bin'], function () {
         .pipe(gulp.dest('bin/mac'));
 });
 
+// Linux bin
+gulp.task('clean-lin-bin', function (callback) {
+    del('bin/lin', callback);
+});
+
+gulp.task('build-lin-executables', ['clean-lin-bin'], function () {
```
Comment (inikulin) [inline Gulpfile.js]: Use `linux` instead of `lin`

File: bin/lin/find-window.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```
Comment (inikulin) [inline bin/lin/find-window.sh]: `bin/linux`

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
`execSync` was added node v0.11.12, that's why tests fail

File: bin/fix-permissions.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+var os       = require('os');
+var execSync = require('child_process').execSync;
+
+// NOTE: It's dangerous to use 'os-family' here on npm<3.0.0: https://github.com/npm/npm/issues/4134
+var platform = os.platform();
+
+if (platform === 'darwin') {
+    execSync('chmod +x ./bin/mac/screenshot');
+    execSync('chmod +x ./bin/mac/generate-thumbnail');
+} else if (platform === 'linux')
```
Comment (georgiy-abbasov) [inline bin/fix-permissions.js]: put the newline, please

**Comment by AndreyBelym** [comment]
FPR

**Comment by georgiy-abbasov** [comment]
thanks for explanation about `xlib`, lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
ping @inikulin

**Comment by inikulin** [comment]
lgtm

**Pull request merged**